pygit2/repository.py

```diff
@@ -291,6 +291,33 @@ def create_reference(self, name, target, force=False, message=None):
         return self.create_reference_symbolic(name, target, force,
                                               message=message)
 
+    def refs_iterator(self, references_return_type = 0):
```

Comment:
Please move this to `Repository.references.iterator(...)`.

Reply:
I also updated Repository.references.__iter__(). Let me know if that is not ok